_posts/2023-03-26-Kafka-IaC.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title:  "Kafka Infrastructure as Code(IaC)"
+author: dasasathyan
+categories: [ Platform Engineering, Data, Infrastructure, Kafka ]
+featured: true
+hidden: true
+teaser: Provision Kafka Infrastructure with Juli-Ops, Pulumi, Terraform
+toc: true
+---
+
+# Infrastructure as Code(IaC)
+
+Organizations need to automate infrastructure provisioning. On the other hand, those tools can’t be too restrictive on development teams. Development teams should have as much autonomy as possible because that's usually how developers get their best work done. It is not easy to manage Infrastructure as going wrong in any of the steps worsens the situation. Also, replicating the same set of configurations for an additional cluster is quite tedious. Here comes the Infrastructure as Code(IaC) handy. It helps us to provision the same Infrastructure across environments with the help of a source code. The advantages of using an IaC are
```

Comment:
Please formulate the sentences little better. You can probably use ChatGPT's help in writing definitions and advantages.